docs/fundamentals/read-operations.txt

```diff
@@ -175,6 +176,124 @@ Use the following syntax to run a find operation that matches all documents:
    more information about ``take()``, see the :ref:`laravel-modify-find` section of this
    guide.
 
+.. _laravel-retrieve-text-search:
+
+Search Text Fields
+------------------
+
+You can perform a text search by using the :manual:`$text
+</reference/operator/query/text>` operator followed
+by the ``$search`` field in your query filter that you pass to the
+``where()`` method. The ``$text`` operator performs a text search on the
+text-indexed fields. The ``$search`` field specifies the text to search for.
+
+A text search retrieves documents that contain a **term** or a **phrase** in the
+text-indexed fields. A term is a sequence of characters that excludes
+whitespace characters. A phrase is a sequence of terms with any number
+of whitespace characters.
+
+.. note::
+
+   Before you can perform a text search, you must create a :manual:`text
+   index </core/indexes/index-types/index-text/>` on
+   the text-valued field. To learn more about creating
+   indexes, see the :ref:`laravel-eloquent-indexes` section of the
+   Schema Builder guide.
+
+After building your query with the ``where()`` method, chain the ``get()``
+method to retrieve the query results.
+
+This example calls the ``where()`` method on the ``Movie`` Eloquent model to
+retrieve documents in which the ``plot`` field contains the phrase
+``"love story"``:
```

Comment:
Issue:

I think the creation of the index needs to be mentioned or shown to understand that "plot" is the indexed field.

Suggestion:
I think either mentioning that the text index was created on the "plot" field or showing an example of creating the index similar to [this Java documentation](https://www.mongodb.com/docs/drivers/java/sync/current/fundamentals/crud/read-operations/text/#text-index) would help a reader understand the connection.

Reply:
I mentioned this in the note, but I can also add a sentence that explains that the example assumes a text index on the `plot` field